doxia-integration-tools/src/main/resources/default-site.xml

```diff
@@ -22,6 +22,11 @@ under the License.
   <bannerLeft>
     <name>${project.name}</name>
   </bannerLeft>
+  <skin>
+    <groupId>org.apache.maven.skins</groupId>
+    <artifactId>maven-default-skin</artifactId>
+    <version>1.3</version>
+  </skin>
```

Comment:
What will be the result - if  project / module doesn't have site.xml but parent of project has with skin defined.

Reply:
Inheritance will kick in. Default site will only be used if upto the root parent no site descriptor has been defined.